Sources/FCM/FCMApnsConfig/FCMApnsApsObject.swift

```diff
@@ -79,7 +79,7 @@ public class FCMApnsApsObject: Codable {
         return FCMApnsApsObject(alertString: nil, sound: "default")
     }
     
-    public convenience init(alertString: String?,
+    public init(alertString: String?,
                             badge: Int? = nil,
```

Comment:
Could you move `badge`, `sound`, `contentAvailable` a little bit left to look good as it was with `convenience` keyword before 🙂

Reply:
fixed in ea45a4c